### PR TITLE
Configuring multiple zones without replication

### PR DESCRIPTION
**Pull request by hmaheswa**
This PR tests multisite scenario of configuring multiple zones without replication
refer this doc: https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/6/html-single/object_gateway_guide/index#configuring-multiple-zones-without-replication_rgw

polarion id: https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83581229

pass log: http://magna002.ceph.redhat.com/ceph-qe-logs/Hemanth_Sai/multiple_zones_without_replication/cephci-run-F7NG0P/

updated pass log: http://magna002.ceph.redhat.com/ceph-qe-logs/Hemanth_Sai/multiple_zones_without_replication/cephci-run-0LYWML/

sample pass log for f"io_info_{os.path.basename(config_file_name)}": http://magna002.ceph.redhat.com/ceph-qe-logs/Hemanth_Sai/multiple_zones_without_replication/cephci-run-6S0N4B/

sample logs for failure scenarios if verify io fails:
http://magna002.ceph.redhat.com/ceph-qe-logs/Hemanth_Sai/multiple_zones_without_replication/cephci-run-2F2JGC/
http://magna002.ceph.redhat.com/ceph-qe-logs/Hemanth_Sai/multiple_zones_without_replication/cephci-run-UYY4P8/

test_byte_range.py testcase is added for creating users, buckets and objects. because it doesnot have sync status check, does not delete users and buckets, populates io_info file and works fine on both sites.

one issue seen with log file not being created after running test.
PR raised for this isssue: https://github.com/red-hat-storage/ceph-qe-scripts/pull/524

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/reef/rgw/tier-2_rgw_multisite_scenarios.yaml [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+              - "yum install -y jq"
+              - "radosgw-admin zonegroup get --rgw-zonegroup=shared > /tmp/zonegroup_shared_backup.json"
+              - "jq -r '.zones[].log_meta=false | .zones[].log_data=false | .zones[].sync_from_all=false' /tmp/zonegroup_shared_backup.json > /tmp/zonegroup_shared.json"
+              - "radosgw-admin zonegroup set --rgw-zonegroup=shared --infile=/tmp/zonegroup_shared.json"
+              - "radosgw-admin period update --rgw-realm india --commit"
+              - "sleep 20"
+      desc: Disabling multisite sync between primary and secondary zones
+      module: exec.py
+      name: disable multisite sync between zones
+      # polarion-id: CEPH-10362
```
Comment (ckulal) [inline suites/reef/rgw/tier-2_rgw_multisite_scenarios.yaml:253]: please uncomment polarian id

File: suites/reef/rgw/tier-2_rgw_multisite_scenarios.yaml [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+      name: deploy cluster
+
+  - test:
+      abort-on-fail: true
+      clusters:
+        ceph-pri:
+          config:
+            command: add
+            id: client.1
+            node: node4
```
[the remark this answers is not in the document]
Comment (hmaheswa) [reply suites/reef/rgw/tier-2_rgw_multisite_scenarios.yaml]: thanks @ckulal , node6 is the client, it is by mistake
this is the conf file: conf/reef/rgw/rgw_multisite.yaml

**Review by ckulal — COMMENTED** [review]
LGTM

**Review by anrao19 — APPROVED** [review]

File: suites/reef/rgw/tier-2_rgw_multisite_scenarios.yaml (hunk 1 above)
Comment (hmaheswa) [reply suites/reef/rgw/tier-2_rgw_multisite_scenarios.yaml]: @ckulal updated with polarion id and uncommented it
https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83581229

**Review by mkasturi18 — COMMENTED** [review]
Changes looks good to me

**Review by ckulal — APPROVED** [review]

**Pull request merged**